ideascube/conf/idb_irq_bardarash.py

```diff
@@ -1,5 +1,5 @@
 """Bardarash in Kurdistan, Iraq"""
-from .azraq import *  # noqa
+from .idb_jor_azraq import *  # noqa
```

Comment:
So, hum, how come the tests never spotted this?

Can you try figuring it out, and fix the tests so that they would fail with this bad import?